mock/db.json

```
@@ -40,6 +40,15 @@
     }
   ],
   "deletionProfiles": [2],
+  "availabilities": [
+    {
+      "id": 1,
+      "startDate": "2023-08-23",
+      "endDate": "2023-12-12",
+      "geographicArea": "Paris, 16ème arrondissement",
+      "category": "Showbiz"
+    }
```

Let's just add a second one to be able to test with multiple availabilities:
```suggestion
    },
    {
       "id": 2,
       "startDate": "2023-03-14",
       "endDate": "2022-12-12",
       "geographicArea": "Paris, 16ème arrondissement",
       "category": "Restauration"
     }
```